ios/Video/RCTVideo.swift

```diff
@@ -1434,7 +1435,11 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
     }
 
     // Continue playing (or not if paused) after being paused due to hitting an unbuffered zone.
-    func handlePlaybackLikelyToKeepUp(playerItem _: AVPlayerItem, change _: NSKeyValueObservedChange<Bool>) {
```

Comment:
I am not sure this will be enough as there are 2 other calls in handleReadyForDisplay and handlePlaybackBufferKeyEmpty.
I think the buffering state shall handled with a local variable to know what is the oldState reported to the UI, and not the old handlePlaybackLikelyToKeepUp

Reply:
Grate Idea! this is working perfect now